ApplicationUnitTests/Commands/Identities/EditIdentityHandlerTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Moq;
+using Identity_Application.Commands.Identities;
+using Identity_Application.Interfaces.Repository;
+using Identity_Application.Interfaces.Authentication;
+using Identity_Domain.Entities.Base;
+using Identity_Application.Models.AppSettingsModels;
+using Microsoft.Extensions.Options;
+using Identity_Application.Models.BaseEntitiesModels;
+
+namespace ApplicationUnitTests.Commands.Identities;
+
+public class EditIdentityHandlerTests
+{
+    private readonly Mock<IGenericRepository<Identity>> _mockIdentityRepository;
+    private readonly Mock<IPasswordHasherService> _mockPasswordHasherService;
+    private readonly Mock<IOptions<PasswordHashSettings>> _mockConfig;
+    private readonly EditIdentityHandler _handler;
+
+    public EditIdentityHandlerTests()
+    {
+        _mockIdentityRepository = new Mock<IGenericRepository<Identity>>();
+        _mockPasswordHasherService = new Mock<IPasswordHasherService>();
+        _mockConfig = new Mock<IOptions<PasswordHashSettings>>();
+        _handler = new EditIdentityHandler(_mockIdentityRepository.Object, _mockPasswordHasherService.Object, _mockConfig.Object);
+    }
+
+    [Fact]
+    public async Task Handle_ValidCommand_ShouldUpdateIdentity()
+    {
+        // Arrange
+        var identityId = 1;
+        var updatedUsername = "UpdatedUsername";
+        var updateIdentityCommand = new UpdateIdentityCommand(identityId, new IdentityVM { Username = updatedUsername, /* other properties */ });
+        var existingIdentity = new Identity { Id = identityId, Username = "OriginalUsername" /* other properties */ };
+
+        _mockIdentityRepository.Setup(repo => repo
+                               .GetAsync(null, null, ""))
+                               .ReturnsAsync(new List<Identity> { existingIdentity });
+
+        _mockIdentityRepository.Setup(repo => repo
+                               .UpdateAsync(It.IsAny<Identity>()));
+
+        // Act
+        await _handler.Handle(updateIdentityCommand, new CancellationToken());
```

Comment:
`new CancellationToken()` - don't use that 
use this `CancellationToken.None`

and use it in each place, where you pass CancellationToken in tests

Reply:
Ok, thanks

Reply:
fixed